src/State/Scope.php

```diff
@@ -96,6 +102,33 @@ public function getTags(): array
         return $this->tags->toArray();
     }
 
+    /**
+     * Sets a new tag in the contexts context.
+     *
+     * @param string $key   The key that uniquely identifies the context
+     * @param array  $value The value
+     *
+     * @return $this
+     */
+    public function setContext(string $key, array $value): self
+    {
+        $this->contexts[$key] = (object) $value;
```

Comment:
You can

Reply:
https://www.php.net/manual/en/language.types.object.php

Reply:
```php
<?php
$obj = (object) 'ciao';
echo $obj->scalar;  // outputs 'ciao'
?>
```

:)

Reply:
I didn't say you can't, I said you shouldn't. That representation doesn't benefit of a lot of performance improvements due to how arrays are handled; so the cast is an overhead, with no benefit, it's not idiomatic and it may suffer further performance issues.

Reply:
Oh okay, sorry
I need to handle this another way since Sentry only accepts objects :)

Reply:
@Jean85 I fixed it :)